### PR TITLE
TELCODOCS:764 - Adding known issues for release 1.3.2 of OSC

### PR DESCRIPTION
**Pull request by MirzWeiss**
Version(s): Only 4.12

Issue:
[TELCODOCS-764](https://issues.redhat.com/browse/TELCODOCS-764)

[Link to docs preview](https://54493--docspreview.netlify.app/openshift-enterprise/latest/sandboxed_containers/sandboxed-containers-1.3-release-notes.html#sandboxed-containers-1-3-known-issues)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: sandboxed_containers/sandboxed-containers-1.3-release-notes.adoc [hunk 1]
```diff
@@ -49,7 +49,18 @@ With this release, the SELinux policy is available when the monitor pod is creat
...
 
-* Setting seLinuxOptions.level at the container level isn't taken into account when deciding the labels that are applied to virtiofsd, unlike when set at the POD level. The users cannot modify SELinux security policy for containers through the Kubernetes' security context. (link:https://issues.redhat.com/browse/KATA-1875[*KATA-1875*]) THIS IS NEW STILL NEEDS DRAFT TEXT FROM DEVS AND EDITS
+* When using {sandboxed-containers-first}, if you set SELinux Multi-Category Security (MCS) labels at the container level, the pod fails to start with the following error:
++
+[source,terminal]
+----
+Error: CreateContainer failed: EACCES: Permission denied: unknown
+----
++
+MCS labels set at the container level aren't applied to virtiofsd. The `kata` runtime doesn't have access to the container's security context when creating a sandboxed container.
```
Comment (gkurz) [inline sandboxed_containers/sandboxed-containers-1.3-release-notes.adoc:59]: ... when creating the VM.

File: sandboxed_containers/sandboxed-containers-1.3-release-notes.adoc [hunk 2]
```diff
@@ -49,7 +49,18 @@ With this release, the SELinux policy is available when the monitor pod is creat
...
++
+This means that virtiofsd won't run with the appropriate SELinux label and won't be able to access host files in the VM on behalf of the container. It isn't possible to rely on MCS to isolate files for each container in the VM; all containers can access all files.
```
Comment (gkurz) [inline sandboxed_containers/sandboxed-containers-1.3-release-notes.adoc:61]: ... to access host files on behalf of the containers running inside the VM.

**Comment by cpmeadors** [comment]
Looks good.  QE approved.

**Review by gkurz — APPROVED** [review]
LGTM. Thanks @MirzWeiss !

**Review by cpmeadors — APPROVED** [review]

File: sandboxed_containers/sandboxed-containers-1.3-release-notes.adoc [hunk 3]
```diff
@@ -49,6 +49,19 @@ With this release, the SELinux policy is available when the monitor pod is creat
...
 == Known issues
 
+* When using {sandboxed-containers-first}, if you set SELinux Multi-Category Security (MCS) labels at the container level, the pod fails to start with the following error:
++
+[source,terminal]
+----
+Error: CreateContainer failed: EACCES: Permission denied: unknown
+----
++
+MCS labels set at the container level aren't applied to virtiofsd. The `kata` runtime doesn't have access to the container's security context when creating the VM.
```
Comment (skrthomas) [inline sandboxed_containers/sandboxed-containers-1.3-release-notes.adoc:59]:
```suggestion
MCS labels set at the container level are not applied to virtiofsd. The `kata` runtime does not have access to the container's security context when creating the VM.
```

File: sandboxed_containers/sandboxed-containers-1.3-release-notes.adoc [hunk 4]
```diff
@@ -49,6 +49,19 @@ With this release, the SELinux policy is available when the monitor pod is creat
...
++
+This means that virtiofsd won't run with the appropriate SELinux label and won't be able to access host files on behalf of the containers in the VM; all containers can access all files in the VM.
```
Comment (skrthomas) [inline sandboxed_containers/sandboxed-containers-1.3-release-notes.adoc:61]:
```suggestion
This means that virtiofsd won't run with the appropriate SELinux label and cannot access host files on behalf of the containers in the VM; all containers can access all files in the VM.
```

**Review by skrthomas — COMMENTED** [review]
Looks good, a few cases of using contractions that need to be corrected.

**Comment by MirzWeiss** [comment]
/label merge-review-needed

However, if it's good to go, **do not merge yet**, please! This is slated for release on January 24th. I just need the merge review done so it can be merged on the release date. I'll request the actual merge separately.

**Pull request merged**